News-Api/src/api.js

```diff
@@ -4,12 +4,17 @@ const newsApi = axios.create({
   baseURL: "https://news-api-ney0.onrender.com/api",
 });
 
-export const getArticles = () => {
-  return newsApi.get("/articles").then((res) => {
+export const getArticles = (topic) => {
+  console.log(topic)
```

Comment:
stray console log

Reply:
Good to see the axios requests extracted into their own folder - nice and reusable